### ISSUE TITLE
Ursa Major appears distorted

### ISSUE THREAD

**Issue by mtekman** [sky-map-team/stardroid#329]
**Summary:** 

Ursa Major looks weird compared to other star maps

**Steps to reproduce:** 

Scroll to Ursa Major, and compare it with the constellations shown in stellarium

**Expected behavior:** 

![Screenshot_2020-07-13 Stellarium Web Online Star Map](https://user-images.githubusercontent.com/20641402/87321226-42501980-c52c-11ea-9611-2acf8bf9437d.png)

**Observed behavior:** 

![Screenshot_20200713-171305_Sky_Map](https://user-images.githubusercontent.com/20641402/87321264-509e3580-c52c-11ea-920b-715feb5b96c1.jpg)

**Device and Android version:** 

One Plus 5 (A5000),  Android 9, Lineage 16.0-20200608, Kernel 4.4.153-lineage

**Screenshots:** 

(see above)

**Comment by mtekman** [comment]
(Btw, thank you for this amazing open source app in F-Droid!)

**Comment by jaydeetay** [comment]
Yes, I guess it is a little strange looking.  If you check the help page you can find the original star maps that were the source of the constellation lines we used 12 years ago.  You'll see that Ursa Major looks a little unusual there too.  Cassiopeia is also drawn in a way that isn't commonly seen - we changed that one to the more familiar W .  I don't plan to redo the constellation lines just now - there are too many other higher priority tasks - but I'll keep this open in case I get a chance in the future.
